docs/src/app/components/pages/customization/themes.jsx

```diff
@@ -30,7 +30,7 @@ const {
   Toggle,
 } = mui;
 
-const {StylePropable, StyleResizable} = Mixins;
```

Comment:
I have the exact same chages in my PR :laughing: :laughing: They are byte-wise equal changes so there won't be conflicts :laughing: :laughing: